client/src/components/question/ThirdPartyIcons.js

```diff
@@ -38,12 +38,15 @@ class ThirdPartyIcons extends React.Component {
         if (index !== null) {
           hiddenId += '-' + index;
         }
-        let answer = document.querySelector(hiddenId);
+		//answer is a span with innerHTML set to the value we're interested in
+        let answer = document.querySelector(hiddenId).innerHTML;
         answer.select();
         document.execCommand('copy');
         this.props.dispatch(Notifications.success({
           title: "Answer copied to clipboard!",
-          message: `Copied "${query}."`,
+		  //replace ' and "
+		  //taken from https://stackoverflow.com/questions/2351576/replacing-quotation-marks-in-javascript
+          message: `Copied "${query.replace(/'/g, "&apos;").replace(/"/g, "&quot;")}."`,
```

Comment:
nit: This could possibly be an extracted util (I can see this being helpful in many places), but not a big deal.